backend/coins/btc/handlers/handlers.go

```diff
@@ -481,11 +482,15 @@ func (handlers *Handlers) postVerifyAddress(r *http.Request) (interface{}, error
 }
 
 func (handlers *Handlers) getCanVerifyExtendedPublicKey(_ *http.Request) (interface{}, error) {
-	btcAccount, ok := handlers.account.(*btc.Account)
-	if !ok {
-		return nil, errp.New("An account must be BTC based to support xpub verification")
+	switch specificAccount := handlers.account.(type) {
+	case *btc.Account:
+		return specificAccount.CanVerifyExtendedPublicKey(), nil
+	case *eth.Account:
+		// TODO: implement secure verification for ETH addresses
```

Comment:
I suppose since the 'i' button now also displays addresses, not only xpubs, the function should be renamed to something like CanVerifyExtendedPublicKeyOrAddress XD
The change was necessary since eth accounts now also have the 'i' button, but it displays an address, not an xpub

Reply:
Already have CanVerifyAddress

Reply:
Yes, but not a can-verify-address endpoint. 
In that case we would need to distinguish in the frontend in the constructor of signingconfiguration.tsx whether an address or xpub will need to be verified and either call the appropriate endpoint or redo the current /can-verify-extended-public-key endpoint to be more general like I mentioned above.
What are your thoughts about that right now?